addons/color-picker/addon.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "HEX color picker",
+  "description": "Adds hex code input to color pickers.",
+  "tags": ["editor"],
+  "credits": [
+    {
+      "name": "Richie Bendall",
+      "link": "https://github.com/ScratchAddons/ScratchAddons/issues/311#issuecomment-699510743"
```

Comment:
You don't like crediting yourself I see.
@Richienb can you confirm whether you want credit? I think this is your full name

Reply:
@apple502j Since you're building the addon, you additionally need to credit yourself.